go/vt/tabletserver/query_executor.go

```diff
@@ -603,12 +607,12 @@ func (qre *QueryExecutor) fullFetch(conn poolConn, parsedQuery *sqlparser.Parsed
 	return qre.execSQL(conn, sql, true)
 }
 
-func (qre *QueryExecutor) fullStreamFetch(conn *DBConn, parsedQuery *sqlparser.ParsedQuery, bindVars map[string]interface{}, buildStreamComment []byte, callback func(*sqltypes.Result) error) error {
+func (qre *QueryExecutor) fullStreamFetch(conn *DBConn, parsedQuery *sqlparser.ParsedQuery, bindVars map[string]interface{}, buildStreamComment []byte, excludeFieldNames bool, callback func(*sqltypes.Result) error) error {
```

Comment:
No it's not the opposite field, it's a different and complimentary field. I'll merge your change in, then you can review the diff. And I'll squash after your review.